metadata-integration/java/datahub-client/src/main/resources/MetadataChangeProposal.avsc

```diff
@@ -1,14 +1,14 @@
 {
```

Comment:
Not sure what this is from, need to investigate.

Reply:
The avsc shouldn't be getting committed. Maybe your .gitignore got modified or something?

Reply:
Yeah pretty sure we don't want this to be committed